evadb/parser/lark_visitor/_create_statements.py

```diff
@@ -315,3 +315,37 @@ def create_database_engine_clause(self, tree):
                     param_dict = self.visit(child)
 
         return engine, param_dict
+
+
+class CreateApplication:
+    def create_application(self, tree):
+        app_name = None
+        if_not_exists = False
+        engine = None
+        param_dict = {}
+
+        for child in tree.children:
+            if isinstance(child, Tree):
+                if child.data == "if_not_exists":
+                    if_not_exists = True
+                elif child.data == "uid":
+                    app_name = self.visit(child)
+                elif child.data == "create_application_engine_clause":
+                    engine, param_dict = self.visit(child)
+
+        create_stmt = CreateDatabaseStatement(
+            app_name, if_not_exists, engine, param_dict, "Application"
```

Comment:
Maybe we should use an enum type for consistentcy here. I think there are different strings in different files: `application`, `Applications`, and `Application`.

Reply:
Done